### DISCUSSION TITLE
Minor importing UX improvements

### DISCUSSION THREAD

**Discussion by novhack** [Orama-Interactive/Pixelorama#735] [Ideas]
I've been using Pixelorama recently quite a lot and there are a few small things I think could improve UX:
## Importing:
1. `New tab` should be renamed to `New project`. New tab can be mistaken for a new frame.
2. Import dialog should remember the last option. I was importing separate frames of animation and I had to always re-select `New frame`.

**Comment by Oni-93** [comment]
These are great ideas. Another small oddity I found, is on the recent projects menu. At the moment, the list shows the most recent project at the bottom instead of the top.

**Comment by Variable-ind** [comment]
The features requested are now implemented in #754 and #755
